tests/app-login.fg

```diff
@@ -9,7 +9,8 @@ odin.user player1 password1234
 odin.user player2 password1234
 odin.sql.file (module.path.join app-login-test-figure.sql)
 
-GET odin/app /login/app01/ 200
+# TODO: Test GET that serve static file
+# GET odin/app /login/app01/ 200
```

Comment:
Yeah, good point. What should this test look like? I think we need to have something a bit more sophisticated going on in the types passed around in the scripting language for this. No doubt it's also related to the bug I just fixed too.